release/pipelines.snapshot.yml

```diff
@@ -0,0 +1,63 @@
+pipelines:
+  - name: build_artifactory_gradle_plugin_snapshot
+    configuration:
+      runtime:
+        type: image
+        image:
+          auto:
+            language: java
+            versions:
+              - "11"
+    steps:
+      - name: Snapshot
+        type: Bash
+        configuration:
+          inputResources:
+            - name: artifactoryGradlePluginSnapshotGit
+          integrations:
+            - name: entplus_deployer
+        execution:
+          onStart:
+            # Save gradle cache
+            - restore_cache_files gradle_cache $res_artifactoryGradlePluginSnapshotGit_resourcePath/.gradle
+          onExecute:
+            - cd $res_artifactoryGradlePluginSnapshotGit_resourcePath
+
+            # Set env
+            - export CI=true
+            - export JFROG_BUILD_STATUS=PASS
+            - export JFROG_CLI_BUILD_NAME=ecosystem-artifactory_gradle_plugin-dev
+            - export JFROG_CLI_BUILD_NUMBER=$run_number
+            - export JFROG_CLI_BUILD_PROJECT=ecosys
+
+            # Configure JFrog CLI
+            - curl -fL https://install-cli.jfrog.io | sh
+            - jf c rm --quiet
+            - jf c add internal --url=$int_entplus_deployer_url --user=$int_entplus_deployer_user --password=$int_entplus_deployer_apikey
+            - jf gradlec --use-wrapper --repo-resolve ecosys-maven-remote --repo-deploy ecosys-oss-snapshot-local --deploy-maven-desc
+
+            # Run audit
+            - jf audit --fail=false
+
+            # Delete former snapshots to make sure the release bundle will not contain the same artifacts
+            - jf rt del "ecosys-oss-snapshot-local/com/jfrog/buildinfo/*" --quiet
+
+            # Run test, build and publish snapshot
+            - >
+              env -i PATH=$PATH HOME=$HOME
+              JFROG_CLI_BUILD_NAME=$JFROG_CLI_BUILD_NAME
+              JFROG_CLI_BUILD_NUMBER=$JFROG_CLI_BUILD_NUMBER
+              JFROG_CLI_BUILD_PROJECT=$JFROG_CLI_BUILD_PROJECT
+              jf gradle clean build artifactoryPublish
+            - jf rt bag && jf rt bce
+            - jf rt bp
+
+            # Distribute release bundle
+            - jf ds rbc ecosystem-artifactory_gradle_plugin-snapshot $run_number --spec=./release/specs/dev-rbc-filespec.json --sign
+            - jf ds rbd ecosystem-artifactory_gradle_plugin-snapshot $run_number --site="releases.jfrog.io" --sync
+
+          onComplete:
+            # Show tests in the *Tests* tab
+            - save_tests $res_artifactoryGradlePluginSnapshotGit_resourcePath/build/test-results/test/
```

Comment:
Are you sure that there are test results under `build/test-results/test/`?

Reply:
no, probably a mistake. removed